common/persistence/cassandra/cassandraHistoryPersistence.go

```diff
@@ -463,12 +463,13 @@ func (h *cassandraHistoryV2Persistence) GetHistoryTree(
 	}
 	query := h.session.Query(v2templateReadAllBranches, treeID)
 
+	pageSize := 100
```

Comment:
Does this need to be configurable or passed here as an argument?

Reply:
not according to the current use case